File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.go
func maxProfit(prices []int) int {
left, right := 0,1
maxP := 0
for right < len(prices){
    if prices[right] > prices[left]{
          res := prices[right] - prices[left]
            if res > maxP{
                maxP = res
            }
    }else{
        left = right
    }
    right+=1
}
return maxP
}

File: Data Structures & Algorithms/count-good-nodes-in-binary-tree/submission-1.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def goodNodes(self, root: TreeNode) -> int:
        res = []
        def dfs(root,maxval):
            if not root:
                return
            if root.right:
                dfs(root.right,max(root.val,maxval))
            if root.left:
                dfs(root.left,max(root.val,maxval))

            if maxval <= root.val:
                res.append(maxval)

        dfs(root,float('-inf'))
        return len(res)